data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+				accept
+			label death
+			`	The leaders gasp, and then, in a moment that seems to last forever, there is silence whilst everyone is looking at you. The silence is shattered when the guards start shooting. They probably did not intend to kill you outright, but apparently they see you as enough of a threat that the result is the same.`
+				die
+		"reputation: Hai" >?= 1
+		log `Convinced the Hai to stop their conflict with me, as request by an Unfettered elder.`
+
+
+
+mission "Unfettered: Bring Hai"
+	name `Bring True Hai back`
+	description `Bring <bunks> Hai from <stopovers> to <origin>.`
+	source "Firelode"
+	stopover "Snowfeather"
+	passengers 10
+	blocked "You will need room for 10 passengers in order to take the next Unfettered mission."
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+		or
+			not "Unfettered: offered bring hai"
+			random < 20
+	on defer
+		set "Unfettered: offered bring hai"
+	on offer
+		conversation
+			`As you exit your ship you notice a group of Unfettered that seem to be discussing something of importance in a big, yet simple, square-shape stadium nearby. A few Unfettered are already heading their way to listen, it does not seem to be restricted of access.`
+			choice
+				`	(Go see them.)`
+				`	(Ignore them.)`
+					defer
+			`	As you head towards them one separates from the rest and comes to meet you. He seems to be lost in his thought and is mumbling on, something about, "We do not get alien visitors often here. There has to be a reason for your presence..." When he speaks again, directly to you, it is with more focus, "Sorry, I was just thinking out loud! Well you are here anyway so maybe you could help us?"`
+			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help a friend, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he says, apparently a bit desperate, considering he knows very little if anything at all about you.`
+			choice
+				`	"Of course, what do you need me to do?"`
+					goto action
+				`	"What do you mean, brother and sisters from Hai space? Are you not different?"`
+					goto brethren
+				`	"Why do you need me to do it?"`
+				`	"This seems like a bad idea, ask someone else."`
+					decline
+			
+			label whyme
+			`	"Well normally they would be able to pay to come here, but the price is pretty high; too high. The Hai make it difficult for us to go anywhere; as you have probably noticed they have a fleet at the edge of our system. And not many Hai captains are willing to bring them here, or at least they ask for a big fee, because of the obvious risks involved, like getting killed in all that cross fire. Yet you somehow made your way here, probably also under fire from our own people, but now that we are allied, you hopefully should not have any issues getting out of here. So, are you willing to help us?"`
```

Comment:
This clashes somewhat with the content introduced with #6500 (Hai merchants) which introduces Merchant Sympathisers who can fairly safely slip through. Of course, they could be busy, or otherwise preoccupied, or perhaps too many sympathisers have recently turned back to peace and they need to recruit some more - i.e. you.

Reply:
The approach here is to say it costs a lot given not everyone is a sympathiser, hence the `for a big fee`, but I could add that they are lacking enough for the big amount of people joining recently, given action is coming.

Reply:
Then that's what needs to be talked about, the increase in demand.

Of course, the Unfettered know that action is coming, but how is that information getting out to the Hai to encourage joiners? It's recent news so if it's getting out enough to lift demand then you probably need to include some content somewhere to make it felt outside of Unfettered space.

Reply:
Perhaps an event to trigger spaceport news in Hai space about rumors of increased activity and recruitment by the Uhai?

Reply:
The Hai sympathisers come to UHai systems, and giving a few JD to the UHai is enough to gain knowledge that they'll attack the aliens north soon, so it doesn't look like that big of a secret.
Yeah we could definitely add that sort of things, and maybe adapt the things Hai and UHai say when you hail them, too